STRUCTURE.md

```diff
@@ -0,0 +1,107 @@
+# Structure
+
+## Cli flow
```

Comment:
Parts of this are incorrect, I can delete it unless @joshstevens19 you'd like to keep this around for future contributors.